priv/endpoints.exs

```diff
@@ -1066,6 +1066,7 @@
           }
         },
         "runtime.sagemaker" => %{
+          "defaults" => %{"credentialScope" => %{"service" => "sagemaker"}},
```

Comment:
This service uses its "parent" service in the credential scope for the authorisation header.

Note that ex_aws is ignoring this value right now, it's only here for documentation purposes.

The only default that has an effect right now is `%{"defaults" => %{"hostname" => hostname}}`, see `ExAws.Config.Defaults.do_host/3`. A good next step would be to honour other defaults that are specified here.

Reply:
This file is autogenerated, so editing it isn't really viable. This kind of mapping will have to go elsewhere.

Reply:
I'll remove this change as the service override in `ExAws.Config.Defaults.defaults(:sagemaker_runtime)` solves the problem of having a different service name in the credential scope.

Good to know that this file is autogenerated. I couldn't find anything in the docs to suggest that. Is this the source? https://github.com/aws/aws-sdk-go/blob/master/models/endpoints/endpoints.json

I'll try moving this specific change upstream.